app/models/manageiq/providers/vmware/infra_manager/vapp.rb

```diff
@@ -0,0 +1,2 @@
+class ManageIQ::Providers::Vmware::InfraManager::Vapp < Vapp
```

Comment:
Or maybe even `OrchestrationStack` as in https://github.com/ManageIQ/manageiq/pull/10062